DOC-3223: Suggestions now display the type of content that was changed (#3890)
tinymce/tinymce-docs: modules/ROOT/pages/8.2.0-release-notes.adoc

// For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].

=== Suggested Edits

The {productname} {release-version} release includes an accompanying release of the **Suggested Edits** premium plugin.

**Suggested Edits** includes the following improvement.

=== Suggestions now display the type of content that was changed
// #TINY-12598

> 
> Previously, the **Suggested Edits** plugin only displayed the word "content" in the suggestion card description, as a generalised context for each suggestion.
> In {release-version}, the **Suggested Edits** plugin displays the type of content as well as the type of edit in the card description, to provide more context for each suggestion.

Reviewer: ```suggestion
Previously, the **Suggested Edits** plugin only displayed the word "content" in the suggestion card description, as a generalised context for each suggestion.

In {release-version}, the **Suggested Edits** plugin displays the type of content as well as the type of edit in the card description, to provide more context for each suggestion.
```
Just adding a space

// For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].

=== Suggested Edits

The {productname} {release-version} release includes an accompanying release of the **Suggested Edits** premium plugin.

**Suggested Edits** includes the following improvement.

> === Operation descriptions now shows the type of content that was changed.

Reviewer 2: ```suggestion
=== Operation descriptions now show the type of content that was changed.
```

> === Operation descriptions now shows the type of content that was changed.

Reviewer: ```suggestion
=== Suggestions now display the type of content that was changed.
```

Author: Happy to update to reflect this change, however we should also update `suggestededits-TINY-12852-2025-10-02.yaml` also.

Reviewer: In the code we call them operations, so that's still fine.

// #TINY-12598

> Previously, the **Suggested Edits** plugin displayed operation cards without indicating the type of content being modified. As a result, reviewers had to open each suggestion individually to determine whether it affected a paragraph, image, table, or another content type.

Reviewer 2: ```suggestion
Previously, the **Suggested Edits** plugin displayed operation cards without indicating the type of content being interacted with. As a result, reviewers had to open each suggestion individually to determine whether it affected a paragraph, image, table, or another content type.
```

This lack of context slowed the review process, particularly in documents with numerous suggestions, as users could not quickly identify or prioritize edits by content type.

> In {release-version}, the **Suggested Edits** plugin now displays contextual descriptions on each operation card. The plugin automatically detects the content type involved and shows descriptive labels such as “added paragraph,” “deleted image,” or “modified table.” Supported content types include paragraphs, headings, images, tables, block quotes, code blocks, horizontal rules, inline text, and more.

Reviewer 2: ```suggestion
In {release-version}, the **Suggested Edits** plugin now displays contextual descriptions on each operation card. The plugin automatically detects the content type involved and shows descriptive labels such as “added paragraph” “deleted image” or “modified table” Supported content types include paragraphs, headings, images, tables, block quotes, code blocks, horizontal rules, inline text, and more.
```

> For information on the **Suggested Edits** plugin, see: xref:suggestededits.adoc[Suggested Edits].

Reviewer: ```suggestion
Previously, the **Suggested Edits** plugin only displayed the word "content" in the suggestion card description, as a generalised context for each suggestion.

In {release-version}, the **Suggested Edits** plugin displays the type of content as well as the type of edit in the card description, to provide more context for each suggestion.

For information on the **Suggested Edits** plugin, see: xref:suggestededits.adoc[Suggested Edits].
```
This is an improvement, so it shouldn't be described as a bug. We also aren't using operation or "modified" to describe suggestions or edits. Edits should refer to the actions made by a user, and suggestions refer to how these edits are displayed in the review.
